src/main/java/edu/cornell/kfs/concur/service/impl/ConcurAccessTokenServiceImpl.java

```diff
@@ -68,6 +115,94 @@ protected ClientRequest buildRefreshAccessTokenClientRequest() {
     }
 
     @Override
+    public void revokeAccessToken() {
```

Comment:
I think we may want to change the revoke Access token to perform:
- revoke current access token &
- request new access token and save it in the database to be used

As far as I understand it, we can request multiple access tokens and the previous one will not be automatically revoked. Because of this I am wondering if it makes sense to even have the request new access token functionality available. Perhaps we could replace the separate request new and revoke with a a single revoke and request new. This can check if a token exists in the DB then revoke and request new. If one does not exists just request new.

I think we want to use the request new access token alone only when generate the token for the first time.
We would only want to use the revoke token when we think the current token may have been compromised or if we want to renew it regularly for security purposes since the refresh functionality does not update the token but just extend it's lifetime. 

The only scenario for using the revoke token alone is if at one point  we are no longer using Concur.